AWS/Copy_ami_to_all_given_AWS_regions.ipynb

```diff
@@ -0,0 +1,317 @@
+{
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "id": "07894245-58bd-475f-b722-8d7513fbe063",
+            "metadata": {
+                "jupyter": {
+                    "source_hidden": false
+                },
+                "name": "Copy AMI to All Given AWS Regions",
+                "orderProperties": [],
+                "tags": [],
+                "title": "Copy AMI to All Given AWS Regions"
+            },
+            "source": [
+                "<img src=\"https://unskript.com/assets/favicon.png\" alt=\"unSkript.com\" width=\"100\" height=\"100\"/> \n",
+                "<h1> unSkript Runbooks </h1>\n",
+                "<div class=\"alert alert-block alert-success\">\n",
+                "    <b> This runbook demonstrates How to copy AMI from one region to another AWS region using unSkript legos.</b>\n",
```

Comment:
This description is not right, you can copy to multiple regions right? not single region.

Reply:
Also destinatio region should be optional, if not provided, we should copy to all enabled AWS regions. To get the list of all available AWS regions, you can use this api [https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-regions.html](this)